README.md

```diff
@@ -87,7 +87,6 @@ More can be found in the docs, but here's a quick list of features which this pr
 - lockScreen()
 - isLocked()
 - shake()
-- complexFind()

```

Comment:
complexfind is still referenced in MobileCommand.java . is this done on purpose?